.github/workflows/main.yml

```diff
@@ -23,12 +23,13 @@ jobs:
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/ 
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - uses: dprint/check@v2.2
+      - run: npm run lint
```

Comment:
where format :/

Reply:
The step above `dprint/check@v2.2` checks formatting.